sensors/test_deps.go

```diff
@@ -119,10 +117,30 @@ func getInvalidSensor() *inject.Camera {
 	cam.ProjectorFunc = func(ctx context.Context) (transform.Projector, error) {
 		return nil, transform.NewNoIntrinsicsError("")
 	}
+	cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
+		return camera.Properties{SupportsPCD: true}, nil
+	}
+	return cam
+}
+
+func getNoPCDCamera() *inject.Camera {
+	cam := &inject.Camera{}
+	cam.NextPointCloudFunc = func(ctx context.Context) (pointcloud.PointCloud, error) {
+		return nil, errors.New("invalid camera")
+	}
+	cam.StreamFunc = func(ctx context.Context, errHandlers ...gostream.ErrorHandler) (gostream.VideoStream, error) {
+		return nil, errors.New("invalid camera")
+	}
+	cam.ProjectorFunc = func(ctx context.Context) (transform.Projector, error) {
+		return nil, transform.NewNoIntrinsicsError("")
+	}
+	cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
+		return camera.Properties{SupportsPCD: false}, nil
+	}
 	return cam
 }
 
-func getFinishedReplaySensor() *inject.Camera {
+func getFinishedReplayLidar() *inject.Camera {
```

Comment:
this is unused right?

Reply:
yes, please remove / comment out with a link to the ticket everything which was added when we thought we would be calling get properties which we are no longer using

Reply:
This was not added by me

Reply:
my mistake, this can stay